pyannote/pipeline/pipeline.py

```diff
@@ -94,7 +94,8 @@ def __setattr__(self, name, value):
 
         If `value` is an instance of `Parameter`, store it in `_parameters`.
         If `value` is an instance of `Pipeline`, store it in `_pipelines`.
-        If `name` is in `_parameters`, store `value` in `_instantiated`.
+        If `name` is in `_parameters`, and `value` isn't an instance of `Parameter`
```

Comment:
Maybe rewrite this as 
```
If `value` ...
elif `value`...
elif `name`...
```
This way, you don't have to clarify "`value` isn't an instance of Parameter"